api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2010,7 +2010,7 @@ public void getOrderTypeByUuid_shouldReturnNullIfNoOrderTypeObjectFoundWithGiven
 	 * @see OrderService#getOrderTypeByClassName(String) 
 	 */
 	@Test
-	public void getOrderTypeByClassName_shouldReturnNullIfNoOrderTypeObjectFoundWithGivenUuid() {
+	public void getOrderTypeByClassName_shouldReturnNullIfNoOrderTypeObjectFoundWithGivenOrderTypeByClassName() {
```

Comment:
The new description of this method seems to be misleading for what it actually does. This calls `getOrderTypeByUuid()`, not `getOrderTypeByClassName()`.

Reply:
Sure thanks , let me have that rectified.What do you however think of the new `getClassName()` method implementations ?

Reply:
I think everything else is actually looking good.